### PR TITLE
Week 6 blog post

### PR DESCRIPTION
**Pull request by m-agour**
image

### CODE REVIEW

File: docs/source/posts/2022/2022-07-25-week-6-mohamed.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+
+- Improved the ``PlaybackPanel`` by adding speed control and the ability to loop the animation. Also, fixed the lagging issue of play and pause buttons and composed them into a single play/pause button.
+
+- Updated the old tutorials' syntax to match the other tutorials and added a new tutorial on position animation using spline interpolation. And added unit tests for the ``PlaybackPanel`` and the newly added color converters in ``colormap.py``.
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-07-25-week-6-mohamed.rst:15]: extra `'` in `tutorials'`

File: docs/source/posts/2022/2022-07-25-week-6-mohamed.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+- Added more hooks to the 2D sliders to cover two more states: ``on_value_changed`` and ``on_moving_slider`` `#634`_.
+
+- Provided the ability to add static actors to the ``Timeline``, which might be needed in the animation part of shivam's glTF project.
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-07-25-week-6-mohamed.rst:19]: maybe `Shivam's`

File: docs/source/posts/2022/2022-07-25-week-6-mohamed.rst (hunk 1 above)
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-07-25-week-6-mohamed.rst:15]: I think these lines should be something like this
`... spline interpolation and added unit tests ...`
or
`... spline interpolation. Added unit tests ...`

**Review by ganimtron-10 — COMMENTED** [review]
Hello @m-agour ,
(Welcome to Week 6 Party 🎉)
Nice Blog! I had a few suggestions mentioned below, PTAL.
Thanks!

File: docs/source/posts/2022/2022-07-25-week-6-mohamed.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+  - If an ``actor`` is added to the ``Timeline`` as a static actor, it won't be animated by the ``Timeline``, but it will get added to the scene along with the ``Timeline`` when the ``Timeline`` is added to the scene.
+
+- Implemented a custom evaluator for the ``Timeline``'s properties.
+
+  - A custom evaluator uses a user-provided function that takes time as input and evaluates the property at that time. This feature is yet to be discussed more in today's meeting.
+
+- Fixed camera rotation, and view-up issue when interacting with the scene.
```
Comment (xtanion) [inline docs/source/posts/2022/2022-07-25-week-6-mohamed.rst:27]: `,` not needed after rotation

File: docs/source/posts/2022/2022-07-25-week-6-mohamed.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+- Improved the ``PlaybackPanel`` by adding speed control and the ability to loop the animation. Also, fixed the lagging issue of play and pause buttons and composed them into a single play/pause button.
+
+- Updated the old tutorials' syntax to match the other tutorials and added a new tutorial on position animation using spline interpolation. And added unit tests for the ``PlaybackPanel`` and the newly added color converters in ``colormap.py``.
+
+- Added more hooks to the 2D sliders to cover two more states: ``on_value_changed`` and ``on_moving_slider`` `#634`_.
+
+- Provided the ability to add static actors to the ``Timeline``, which might be needed in the animation part of shivam's glTF project.
+
+  - If an ``actor`` is added to the ``Timeline`` as a static actor, it won't be animated by the ``Timeline``, but it will get added to the scene along with the ``Timeline`` when the ``Timeline`` is added to the scene.
```
Comment (xtanion) [inline docs/source/posts/2022/2022-07-25-week-6-mohamed.rst:21]: This sentence is too long. (for e.g. You can remove `when the ``Timeline`` is added to the scene` and it won't make much difference in the sentence)

**Review by xtanion — COMMENTED** [review]
Hi @m-agour, Nice blog. I have a few small suggestions PTAL.
Thanks

**Comment by m-agour** [comment]
Thanks @ganimtron-10 and @xtanion so much for reviewing my blog post.

File: docs/source/posts/2022/2022-07-25-week-6-mohamed.rst [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+
+What did you do this week?
+--------------------------
+
+- Improved the ``PlaybackPanel`` by adding speed control and the ability to loop the animation. Also, fixed the lagging issue of play and pause buttons and composed them into a single play/pause button.
+
+- Updated the old tutorials' syntax to match the other tutorials and added a new tutorial on position animation using spline interpolation. Added unit tests for the ``PlaybackPanel`` and the newly added color converters in ``colormap.py``.
+
+- Added more hooks to the 2D sliders to cover two more states: ``on_value_changed`` and ``on_moving_slider`` `#634`_.
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-07-25-week-6-mohamed.rst:17]: What is difference? why do you need to add them? can you explain more here

File: docs/source/posts/2022/2022-07-25-week-6-mohamed.rst [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+- Provided the ability to add static actors to the ``Timeline``, which might be needed in the animation part of Shivam's glTF project.
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-07-25-week-6-mohamed.rst:19]: > Shivam's glTF project

add a link

File: docs/source/posts/2022/2022-07-25-week-6-mohamed.rst [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+- Implemented a custom evaluator for the ``Timeline``'s properties.
+
+  - A custom evaluator uses a user-provided function that takes time as input and evaluates the property at that time. This feature is yet to be discussed more in today's meeting.
+
+- Fixed camera rotation and the view-up issue when interacting with the scene.
+
+
+What is coming up next week?
+----------------------------
+Next week's work is yet to be determined.
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-07-25-week-6-mohamed.rst:32]: it seems you can update now

**Comment by skoudoro** [comment]
Thank you for the update @m-agour, merging

**Pull request merged**